week-2/13-order/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Flexbox</title>
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/week2.css">
+	<link rel="stylesheet" href="/css/week2-flexbox.css">
+  <link rel="stylesheet" href="flexbox.css">
+  <link rel="stylesheet" href="./ignore_this_folder/custom.css">
+</head>
+
+<body>
+  <div class="site-wrapper site-wrapper">
+    <div class="site-header">
+      <h1 class="site-header__title">First</h1>
+    </div>
+    <ul class="countries countries--first">
+			<li class="country" id="brazil">Brazil</li>
+			<li class="country" id="croatia">Croatia</li>
+			<li class="country" id="ethiopia">Ethiopia</li>
+			<li class="country" id="laos">Laos</li>
+			<li class="country" id="uganda">Uganda</li>
+    </ul>
+    <p>⬇️⬇️ should look like this ⬇️⬇️</p>
```

Comment:
I like this idea but I think it maybe makes the exercises look a little confusing. Instead of putting the result right next to the exercise in each case, maybe we can put the result below the exercises in a separate panel?

![selection_062](https://user-images.githubusercontent.com/2306629/50685737-4b485c80-1012-11e9-8838-b4b64a8c63af.png)

(This image also shows the Ubuntu font, which most students will see since they'll be on Linux.)

Reply:
Thanks a lot for building the preview image! 

I can see how the current setup can make it confusing. I don't know if it should be at bottom though - ideally, I was looking to avoid making the student scroll back and fourth between the solution and their implementation while working on each part.

I'll try having the solution in a separate panel, but side by side rather than the bottom. We can then discuss how it looks.